Dockerfile

```diff
@@ -33,4 +33,4 @@ RUN apt-get update && apt-get install jq -y
 
 RUN chmod +x entrypoint.sh
 #ENTRYPOINT ["/CarbonAwareCLI"]
-ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Why was this changed as part of the documentation update?  Is is part of a broader change?

Please either:
* Add a description to the PR overview _or_ 
* Remove from the PR as it does not relate to the current PR documentation (preferred).

While it's a small change a general documentation clarification change shouldn't change behaviour (even if small)

Reply:
I do not intent to change any other files - it seems to be caused by merge commit.  
I reverted the change in Dockerfile. Could you review again?